java-common-protos/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,51 +0,0 @@
----
```

Comment:
We didn't have an issue template before for gapic-generator-java, however, this is a good practice we should start to adopt. We should come up with a generic template that work for all submodules, it could be improved in a separate PR though.